addons/sale/views/sale_portal_templates.xml

```diff
@@ -432,6 +432,10 @@
                             t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'/>
                     </div>
                 </div>
+                <div t-if="sale_order.client_order_ref">
```

Comment:
Shouldn't it be in the above row ? Here you add content between rows, meaning it will probably be badly incorporated in the grid ...

Reply:
Hello @tde-banana-odoo 

Changes are done.
Can you please check it.
Thanks :)

Reply:
But you are still adding content in a bootstrap grid without respecting columns ... did not test but I feel like you are going to have issues with mobile, ... as all elements are correctly using row / col, except your addition.

Reply:
Hello @tde-banana-odoo 

Now i have created my own row so i think the view will not break.
Thanks :)